app/views/pets/edit.html.erb

```diff
@@ -1,15 +1,29 @@
 <h2>Update Pet</h2>
 
-<%= form_tag "/pets/#{@pet.id}", method: :patch do %>
-    <%= label_tag :name %>
-    <%= text_field_tag :name, "#{@pet.name}" %>
-    <%= label_tag :image %>
-    <%= text_field_tag :image, "#{@pet.image}" %>
-    <%= label_tag :description %>
-    <%= text_field_tag :description, "#{@pet.description}" %>
-    <%= label_tag "Approximate age" %>
-    <%= text_field_tag :age, "#{@pet.age}" %>
-    <%= label_tag :sex %>
-    <%= text_field_tag :sex, "#{@pet.sex}" %>
-    <%= submit_tag "Update Pet" %>
-<% end %>
\ No newline at end of file
+<div class="input-form">
+    <%= form_tag "/pets/#{@pet.id}", method: :patch do %>
+        <div class="row form-entry">
+            <%= label_tag :name, nil, class: "col-3 light-text" %>
+            <%= text_field_tag :name, "#{@pet.name}", class: "col-8", required: true %>
+        </div>
```

Comment:
Each form field has its own class now, these are all using the row class in order to get them inline and allow for easier customization. This same system is used on all the forms on the site.